canonical-websites/www.ubuntu.com
# updated subiquity message on /download/server and redirect

## Done

- redirected - community/get-involved/ to https://discourse.ubuntu.com/t/contribute/26
- updated text on /download/server per the [copy doc](https://docs.google.com/document/d/1FFQQUtzFGH88hA30qnZm-3pwVqcVl9UYyrNT4qQWrSM/edit)


## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- see that you are redirected from http://0.0.0.0:8001/community/get-involved/ to https://discourse.ubuntu.com/t/contribute/26
- see that http://0.0.0.0:8001/download/server text about subiquity matches the copy doc "This release uses our new installer, Subiquity.  If you need support for options not implemented in Subiquity, such as encrypted filesystem support, the traditional installer can be found on the alternative downloads page."

## Issue / Card

- server page from slangasek
- redirect from infinity
